assets/css/ccore.scss

```diff
@@ -7,14 +7,27 @@
 
 .ccore-home {
   background-image: url("/assets/images/background-home.png");
-  background-size: contain;
+  background-size: 300%;
 }
 
 .ccore-docs {
   background-image: url("/assets/images/background-docs.png");
-  background-size: contain;
+  background-size: 300%;
 }
 
+@include mq($breakpoint-md) {
+  .ccore-home {
+    background-image: url("/assets/images/background-home.png");
```

Comment:
Should we use `.svg`'s?

Reply:
Not for Background images. It makes Chrome (at least) lag really badly when scrolling.

Reply:
Ahh ok!